services/web-app/components/catalog-item/catalog-item.js

```diff
@@ -110,7 +110,8 @@ CatalogItemContent.propTypes = {
 
 const CatalogItemMeta = ({ asset, className, properties }) => {
   const renderStatus = () => {
-    const { name } = status[asset.content.status]
+    const statusKey = asset?.content.status.key ?? asset?.content.status ?? 'draft'
+    const { name } = status[statusKey]
```

Comment:
I'd be more comfortable with
```suggestion
    const statusKey = asset?.content.status?.key ?? asset?.content.status ?? 'draft'
    const { name } = status[statusKey]
```
if there's an idea that it could get to the 'draft' part then status is probably null or undefined, which means we'd hit an error when we try to evaluate `asset?.content.status.key` the first time around